### PR TITLE
base os changes for twitter and movielens datasets.

### PR DESCRIPTION
**Pull request by VamshiMuthineni**
https://github.com/parsa-epfl/cloudsuite/issues/259.
do not merge until twitter dataset link is also updated.
---------------------
Updated. You can review and merge now.

### CODE REVIEW

File: datasets/twitter-dataset-graph/Dockerfile [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 # Install packages
 RUN BUILD_DEPS="wget unzip" \
     && apt-get update && apt-get install -y --no-install-recommends ${BUILD_DEPS} \
     && rm -rf /var/lib/apt/lists/* \
     && mkdir -p /data \
-    && wget --no-check-certificate -O /data/Twitter-dataset.zip socialcomputing.asu.edu/uploads/1296759055/Twitter-dataset.zip \
+    && wget --no-check-certificate -O /data/Twitter-dataset.zip https://cloudsuite.ch/download/graph_analytics/twitter_dataset.zip \
     && unzip -d /data /data/Twitter-dataset.zip \
     && rm /data/Twitter-dataset.zip \
-    && sed -i "s/,/ /g" /data/Twitter-dataset/data/edges.csv \
```
Comment (neo-apz) [inline datasets/twitter-dataset-graph/Dockerfile:11]: Why is this removed?

Comment (VamshiMuthineni) [reply datasets/twitter-dataset-graph/Dockerfile:11]: the data given to you is already space separated. Earlier we were parsing comma separated to space but now it's not necessary.

File: datasets/twitter-dataset-graph/Dockerfile [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
     && mv /data/Twitter-dataset/data/edges.csv /data/edges.csv \
```
Comment (neo-apz) [inline datasets/twitter-dataset-graph/Dockerfile:11]: I believe this path is incorrect. There is no directory called `data` inside the zip file.

Comment (VamshiMuthineni) [reply datasets/twitter-dataset-graph/Dockerfile]: updated.
